Assignment01/step-by-step-java.md

```diff
@@ -15,23 +15,23 @@ As a reminder, this is how the services will interact with each other:
 
 ## Step 1. Run the VehicleRegistration service
 
-1. Open the `java` folder in VS Code. This guide assumes VS Code, but feel free to use an editor or IDE you're comfortable with.
+1. Open the source-code folder in VS Code. This guide assumes VS Code, but feel free to use an editor or IDE you're comfortable with.
 
    > Throughout the assignment you can execute all steps in the same instance of editor or IDE window.
 
 1. Open a terminal window.
 
    > You can do this by using the hotkey ``Ctrl-` `` (Windows) or ``Shift-Ctrl-` `` (macOS).
 
-1. Make sure the current folder is `java/VehicleRegistrationService`.
+1. Make sure the current folder is `VehicleRegistrationService`.
 
 1. Start the service using `mvn spring-boot:run`.
 
 > If you receive an error here, please double-check whether or not you have installed all the [prerequisites](../README.md#Prerequisites) for the workshop!
 
 Now you can test whether you can call the VehicleRegistrationService. You can do this using a browser, cURL or some other HTTP client. But there is a convenient way of testing RESTful APIs directly from VS Code (this uses the REST Client extension VS Code):
 
-1. Open the file `java/VehicleRegistrationService/test.http` in your editor. The request in this file simulates retrieving the vehicle- and owner information for a certain license-number.
+1. Open the file `VehicleRegistrationService/test.http` in your editor. The request in this file simulates retrieving the vehicle- and owner information for a certain license-number.
```

Comment:
I think the HTTP request files have a different name in the Java track, but it makes sense to always call them **test.http**. Done in [mthmulders/dapr-workshop-java #f4e0befb](https://github.com/mthmulders/dapr-workshop-java/commit/f4e0befbf63a0f434649873d6a45f5b9c5ca59ac).